pkg/sql/gcjob_test/gc_job_test.go

```diff
@@ -317,10 +317,30 @@ SELECT job_id
   FROM [SHOW JOBS]
  WHERE job_type = 'SCHEMA CHANGE GC' AND description LIKE '%foo%';`,
 	).Scan(&jobID)
-	tdb.CheckQueryResultsRetry(t,
-		"SELECT running_status FROM crdb_internal.jobs WHERE job_id = "+jobID,
-		[][]string{{string(sql.RunningStatusWaitingForMVCCGC)}},
-	)
+
+	const expectedRunningStatus = string(sql.RunningStatusWaitingForMVCCGC)
+	testutils.SucceedsSoon(t, func() error {
+		var runningStatus, lastRun, nextRun, numRuns, jobErr string
+		tdb.QueryRow(t, fmt.Sprintf(`
+SELECT running_status, error, last_run, next_run, num_runs
+FROM crdb_internal.jobs
+WHERE job_id = %s`, jobID)).Scan(&runningStatus, &jobErr, &lastRun, &nextRun, &numRuns)
+
+		t.Logf(`Details about SCHEMA CHANGE GC job
```

Comment:
i wouldn't have thought this needed to be broken up into multiple lines, but if you find it more readable, i'm ok with it.

Reply:
Changed it to a single line